### ISSUE TITLE
Machining

### ISSUE THREAD

**Issue by PGR-14** [CleverRaven/Cataclysm-DDA#75067]
### Is your feature request related to a problem? Please describe.

I saw a lathe at a hardware store, & it got me thinking.

### Solution you would like.

Machined items, about as good as the pre-cataclysm variants. I.E. Lever guns, Pump actions, parts, etc. Made using a lathe or CNC

### Describe alternatives you have considered.

A: Just adding the lathe (&/Or alternatives)
B: Not doing it
C: Make the lathe &/or the CNC, but not the machined weapons

**Comment by kevingranade** [comment]
For something to be an issue,  it needs a lot more than a few seconds thought put into it. "Just add a lathe" doesn't actually resolve anyting,  how do we handle tooling, how do we handle feedstock, how do we handle work piece sizing?

We know what lathes are and roughly what they can do,  we don't need reminders of that, we need serious consideration of how to fit them into the game.

Tl;dr 
If your suggestion resembles the picture on the left, please just don't post it.
![raowr6zqaxsy53j8a72w](https://github.com/user-attachments/assets/12281b53-eb64-4a30-8a87-3e4ef989729d)
